### PR TITLE
Fixed an issue in BindingInfo validation

### PR DESCRIPTION
**Pull request by SNikalaichyk**
Fixed an issue in BindingInfo validation that caused multiple bindings with the same port and protocol treated as invalid.

### CODE REVIEW

File: DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1 [hunk 1]
```diff
@@ -43,7 +43,7 @@ VerboseConvertToWebBindingIgnoreBindingInformation = BindingInformation is ignor
 VerboseConvertToWebBindingDefaultPort = Port is not specified. The default "{0}" port "{1}" will be used.
 VerboseConvertToWebBindingDefaultCertificateStoreName = CertificateStoreName is not specified. The default value "{0}" will be used.
 VerboseTestBindingInfoSameIPAddressPortHostName = BindingInfo contains multiple items with the same IPAddress, Port, and HostName combination.
-VerboseTestBindingInfoSamePort = BindingInfo contains multiple items with the same Port.
+VerboseTestBindingInfoSamePortDifferentProtocol = BindingInfo containts items that share the same Port but have different Protocols.
```
Comment (tysonjhayes) [inline DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1]: This should be contains.

File: Tests/Unit/MSFT_xWebsite.Tests.ps1 [hunk 2]
```diff
@@ -1263,13 +1263,13 @@ try
 
             }
 
-            Context 'BindingInfo contains multiple items with the same Port' {
+            Context 'BindingInfo containts items that share the same Port but have different Protocols' {
```
Comment (tysonjhayes) [inline Tests/Unit/MSFT_xWebsite.Tests.ps1]: contains

File: Tests/Unit/MSFT_xWebsite.Tests.ps1 [hunk 3]
```diff
@@ -1236,19 +1236,19 @@ try
 
                 $MockBindingInfo = @(
                     New-CimInstance -ClassName MSFT_xWebBindingInformation -Namespace root/microsoft/Windows/DesiredStateConfiguration -Property @{
-                        Protocol              = 'http'
+                        Protocol              = 'https'
                         IPAddress             = '*'
-                        Port                  = 8080
+                        Port                  = 443
```
Comment (tysonjhayes) [inline Tests/Unit/MSFT_xWebsite.Tests.ps1]: I'm unclear why this needed to change, why are we changing it from 8080 to 443, and adding in a thumbprint etc? Does the test not catch this as is?

Comment (SNikalaichyk) [inline Tests/Unit/MSFT_xWebsite.Tests.ps1]: It did and it does. Just wanted to make this test more real-life.

Comment (tysonjhayes) [inline Tests/Unit/MSFT_xWebsite.Tests.ps1]: None of the changes to the test file seem necessary to fix these issues, if you would like to make the tests "more realisitc" could you please submit it as a seperate PR?

**Comment by SNikalaichyk** [comment]
**BindingInfo** for test case "**Test-BindingInfo**: *BindingInfo contains multiple items with the same IPAddress, Port, and HostName combination*" was updated for the new function behavior.

**Comment by tysonjhayes** [comment]
LGTM. Sorry for not getting back to this sooner.

**Pull request merged**